### PR TITLE
helloworld.cpp added

### PR DESCRIPTION
**Pull request by adityakuppa26**
open for your comments

### CODE REVIEW

**Comment by bprvnrj97** [comment]
Class one

**Comment by LalithRaju** [comment]
good

**Comment by ajinkyakhadilkar** [comment]
nice

**Comment by preetha2907** [comment]
can be made better

**Comment by bprvnrj97** [comment]
I need member function

File: helloworld.cpp [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+class First
+{
+	
+}
+
+class Second
+{
+	public:
+		int var;
```
Comment (LalithRaju) [inline helloworld.cpp]: what is var... please give it a proper name

**Comment by preetha2907** [comment]
change the name of the classes also

File: helloworld.cpp [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+class First
+{
+	
+}
+
+class Second
+{
+	public:
+		int area;
```
Comment (LalithRaju) [inline helloworld.cpp:11]: epic

**Pull request merged**